site/content/en/docs/contributing/development-environment.md

```diff
@@ -73,25 +80,36 @@ description: 'Installing a development environment for different operating syste
   >
   > Read this article [Node Sass does not yet support your current environment](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
 
-- Open new terminal (Ctrl + Shift + T), run Visual Studio Code from the virtual environment
+- Install [Docker Engine](https://docs.docker.com/engine/install/ubuntu/) and [Docker-Compose](https://docs.docker.com/compose/install/)
 
+- Pull OpenPolicyAgent Docker-image (run from CVAT root dir):
   ```sh
-  source .env/bin/activate && code
+  sudo docker-compose -f docker-compose.yml -f docker-compose.dev.yml up cvat_opa
   ```
 
-- Install following VS Code extensions:
+### Run CVAT
+- Start npm UI debug server (run the following command from CVAT root directory):
+    - If you want to run CVAT in localhost:
+        ```sh
+        npm run start:cvat-ui
+        ```
+    - If you want to access CVAT from outside of your host:
+        ```sh
+        CVAT_UI_HOST='<YOUR_HOST_IP>' npm run start:cvat-ui
+        ```
+- Open a new terminal window and run VScode from the virtual environment (run the following command from CVAT root directory):
 
-  - [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
-  - [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
-  - [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
-  - [Stylelint](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint)
-  - [vscode-remark-lint](https://marketplace.visualstudio.com/items?itemName=drewbourne.vscode-remark-lint)
-  - [licenser](https://marketplace.visualstudio.com/items?itemName=ymotongpoo.licenser)
-  - [Trailing Spaces](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces)
+  ```sh
+  source .env/bin/activate && code
+  ```
+
+- Inside VScode, Open CVAT root dir
 
-- Reload Visual Studio Code from virtual environment
+- Select `server: debug` configuration and run it (F5) to run REST server and its workers
+- Make sure that ```Uncaught Exceptions``` option under breakpoints section is unchecked
+- If you choose to run CVAT in localhost: Select `server: chrome` configuration and run it (F5) to open CVAT in Chrome
```

Comment:
There is also `ui.js: debug` configuration with npm background task

Reply:
> There is also `ui.js: debug` configuration with npm background task

It does not work for me. Maybe I am trying to do something wrong.